extensions/amp-consent/0.1/consent-config.js

```diff
@@ -245,19 +244,23 @@ export class ConsentConfig {
   }
 
   /**
-   * Validate if story consent then no promptUiSrc
+   * Validate if story consent then no promptUISrc
    * @param {!JsonObject} config
    * @return {!JsonObject}
    */
   checkStoryConsent_(config) {
-    if (childElementByTag(this.element_, AMP_STORY_CONSENT_TAG)) {
-      userAssert(
-        !config['promptUISrc'],
-        '%s: `promptUiSrc` cannot be specified while using %s.',
-        TAG,
-        AMP_STORY_CONSENT_TAG
-      );
-    }
+    // TODO(wg-stories):
+    // We verify that configurations with an <amp-story-consent> descendant have
+    // no promptUISrc, but we don't verify that the element is present in
```

Comment:
What element? Looking at [this example](https://github.com/ampproject/amphtml/blob/f5c34b95ebfb0384426809607d5db6b40952af6d/examples/amp-consent/cmp-vendors.amp.html#L209-L222) I'm not sure what we should look at.

Reply:
`<amp-story-consent>` is the element.

Your linked example is not a Story. See [`examples/amp-story/consent.html`](https://github.com/ampproject/amphtml/blob/f5c34b95ebfb0384426809607d5db6b40952af6d/examples/amp-story/consent.html), which contains the mentioned element. 

It seems to me like the intention with `<amp-story-consent>` is to ensure a consistent UI for consent prompts in Stories. (Please correct me if this is not accurate.)

However, I noticed that there's no enforcement of an `<amp-story-consent>` element existing inside `<amp-consent>` in Stories, which means that publishers could be using `promptUISrc` instead.

Using `promptUISrc` creates a different UX, which seems like it wasn't intended in the first place.

I'm looping you in to clarify whether this is intended or not.

- If it's not intended to allow `promptUISrc`, you might want to provide a migration path.
- If it's intended to support it, then we should remove this check. We prioritize `promptUI` over `promptUISrc` anyway, so this check does not prevent any undefined behavior.